doc/src/records/rsys.md

```diff
@@ -274,6 +591,17 @@ For rsType=ASHP, auxiliary electric (“strip”) heating capacity. If AUTOSIZEd
   ----------- --------------------------- ------------- -------------- -----------------
   Btu/hr      *AUTOSIZE* or *x* $\ge$ 0   0             no             constant
 
+**rsAFUEAuxH=*float***
+
+Auxiliary furnace heating for annualized fuel utilization efficiency.
```

Comment:
See the previous documentation that you deleted before (from `rsCOPAuxH`). I think that was the same input, but with the wrong name.